.github/workflows/build.yml

```diff
@@ -70,3 +82,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
+          build-args: |
+            BUILD_DATE=${{ steps.args.outputs.BUILD_DATE }}
```

Comment:
Please replace this line with the next one:
`BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')`